### PR TITLE
Remove -nointro for BioShock 2 Remastered and add all missing BioShock titles

### PR DESCRIPTION
**Pull request by kasha695**
The -nointro fix can be removed as this has been addressed - [ValveSoftware/Proton/issues/482](https://github.com/ValveSoftware/Proton/issues/482#issuecomment-780769188)

I have also tested this as working myself by allowing the intro videos to play then loading a game.

Added the 3 missing BioShock titles that all need Esync and Fsync disabled. This is for parity with Valve as they have disabled both of these for the titles as mentioned [here.](https://github.com/ValveSoftware/Proton/issues/482#issuecomment-988954117)

They can be separated if needed but I've done them linked to BioShock 2 Remastered.

### CODE REVIEW

**Comment by Root-Core** [comment]
We don't need to bother if esync and fsync are already handled by Proton, as [Kisak's comment](https://github.com/ValveSoftware/Proton/issues/482#issuecomment-988954117) suggests.

File: gamefixes-steam/409720.py [hunk 1]
```diff
@@ -4,12 +4,7 @@
...
-    # to continue. That screen does not respond to keyboard or mouse,
-    # so there is no way to continue. -nointro disables that screen
-    # (but also the intro's at the start of the game).
-    util.append_argument('-nointro')
-
-    # ESYNC causes texture problems and frequent hangs.
+    """Disable ESYNC and FSYNC"""
+    # ESYNC and FSYNC causes low quality texture problems in all BioShock 1 and 2 versions (Original and Remastered).
     util.disable_esync()
+    util.disable_fsync()
```
Comment (R1kaB3rN) [inline gamefixes-steam/409720.py:10]: Putting aside the user reports, which may be outdated at this time, it’s not clear to me where the fsync and esync issue was addressed for these titles. I’m not seeing it in the proton script nor in Valve’s fork of wine. Are you able to reference where exactly the issue has been addressed.

**Review by R1kaB3rN — CHANGES_REQUESTED** [review]

File: gamefixes-steam/409720.py (hunk 1 above)
Comment (R1kaB3rN) [reply gamefixes-steam/409720.py:10]: To be clear, I’m referring to the commen your sourced:

https://github.com/ValveSoftware/Proton/issues/482#issuecomment-988954117

Comment (R1kaB3rN) [reply gamefixes-steam/409720.py:10]: In fact, if we’re to trust Kisak’s word, we can remove disabling fsync and esync all together. Adding these fixes in implies that this issue still exist and they’re wrong. If so, then this needs to be reported.

Comment (kasha695) [reply gamefixes-steam/409720.py:10]: I will do some further tests to see if these changes are necessary.

Comment (kasha695) [reply gamefixes-steam/409720.py:10]: I've gone ahead and put together a new PR #176 which addresses the necessary fixes.

If that's all good I'll go ahead and close this one.

File: gamefixes-steam/409720.py [hunk 2]
```diff
@@ -4,12 +4,7 @@
 
 
 def main() -> None:
-    """Disable ESYNC, disable intro's"""
-    # After loading the game, or a save file, a key needs to be pressed
-    # to continue. That screen does not respond to keyboard or mouse,
-    # so there is no way to continue. -nointro disables that screen
-    # (but also the intro's at the start of the game).
-    util.append_argument('-nointro')
```
Comment (R1kaB3rN) [inline gamefixes-steam/409720.py:12]: I'm deciding that we should keep the fsync and esync related fixes for those titles while removing the disable intro argument until Kisak is able to substantiate their statement. [I've already submitted a comment to them](https://github.com/ValveSoftware/Proton/issues/482#issuecomment-988954117).

**Review by R1kaB3rN — CHANGES_REQUESTED** [review]

**Review by R1kaB3rN — APPROVED** [review]

File: gamefixes-steam/409710.py [hunk 3]
```diff
@@ -0,0 +1 @@
+409720.py
```
Comment (R1kaB3rN) [inline gamefixes-steam/409710.py:1]: According to the linter, there's a problem with your changes. This patch should fix it:

Comment (R1kaB3rN) [inline gamefixes-steam/409710.py:1]: According to the linter, there's a problem with your changes. This patch should fix it:

[0001-Fix-lint.txt](https://github.com/user-attachments/files/18156137/0001-Fix-lint.txt)

**Review by R1kaB3rN — CHANGES_REQUESTED** [review]

File: gamefixes-steam/409710.py (hunk 3 above)
Comment (kasha695) [reply gamefixes-steam/409710.py]: Fixes have been made, should be good to go.

**Review by R1kaB3rN — APPROVED** [review]

**Pull request merged**